packages/server/lib/crons/exportMetrics.ts

```diff
@@ -0,0 +1,46 @@
+import * as cron from 'node-cron';
+import { errorManager, ErrorSourceEnum, connectionService } from '@nangohq/shared';
+import { stringifyError, getLogger, metrics } from '@nangohq/utils';
+import tracer from 'dd-trace';
+
+const logger = getLogger('Server');
+const cronName = '[exportUsageMetrics]';
```

Comment:
It's legacy from when we didn't have better. Using the logger name will automatically prepend all the logs
```suggestion
const logger = getLogger('Server.exportUsageMetrics');
```